src/components/RichSelect/index.tsx

```diff
@@ -433,17 +472,16 @@ Input.defaultProps = {
   value: '',
 }
 
-const Option = ({ selectProps, value, label, ...props }) => (
+const Option = ({ selectProps, value, label, ...props }: OptionProps<SelectOption, boolean> & SelectOption) => (
   <div
-    data-testid={`option-${selectProps.name}-${
+    data-testid={`option-${selectProps.name || ''}-${
```

Comment:
we have:
```js
selectProps: {
      name = ''
}
```

So name by default value is already `''`
```suggestion
    data-testid={`option-${selectProps.name}-${
```

Reply:
react-select `OptionProps` typing declares `name` as optional, and I prefer to use an empty string as fallback here rather than trying to override types.